### ISSUE TITLE
Cannot craft soft mallet

### ISSUE THREAD

**Issue by HeroOfOlympus** [GTNewHorizons/GT-New-Horizons-Modpack#7680]
Hey, for some reason I cannot craft a wooden soft mallet. Doesn't matter using the same type of wood or multiple ones, also checked by clicking U on the type of wood I was using and then waiting until every material was correct.

**Comment by HeroOfOlympus** [comment]
Actually, I found out that it's every item that can't be put into crafting bench (vanilla) or 2x2 character crafting. I'm using 2.1.0.4 and also the newest 2.1.0-GTNH-beta, my fault cause I forgot to add a label.

**Comment by boubou19** [comment]
is it happening if you disable nei? default keybind is O iirc

**Comment by HeroOfOlympus** [comment]
Eh, I've used a bad word, I meant that it's happening with every item soo I cannot use crafting filling from NEI at all. Sorry for the misunderstanding.

**Comment by Prometheus0000** [comment]
Do you mean with the ? button? But you can put them in manually?

**Comment by HeroOfOlympus** [comment]
Yes exactly, but shadowed one is working but auto inputting to crafting doesn't. Also I couldn't get that worked at all with anything.

**Comment by Prometheus0000** [comment]
Sounds like https://github.com/GTNewHorizons/NotEnoughItems/issues/44 and https://github.com/GTNewHorizons/NotEnoughItems/issues/61.

**Comment by boubou19** [comment]
the explaination is actually in https://github.com/GTNewHorizons/NotEnoughItems/issues/29

**Comment by Prometheus0000** [comment]
Would that affect a mallet though? The items don't have any nbt yet?

@HeroOfOlympus What version are you using?

**Comment by Prometheus0000** [comment]
Nvm I see you already answered that

**Comment by HeroOfOlympus** [comment]
It's definitely the #61. I've checked more and it doesn't actually work at all. I haven't found single item that worked. Tested out in player crafting and vanilla crafting. Tested mallet with single type of wood still didn't worked. I may try to change gamemode to creative or give myself more oredict items to check.

**Comment by Prometheus0000** [comment]
@leagris I know you moved this, but it looks like it belongs in the NEI repo after all.
